src/styles/review.css

```diff
@@ -0,0 +1,54 @@
+
+.card-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 16px;
+}
+
+.card-price__text {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+  margin: 0;
+}
+
+.card-price__value {
+  font-weight: 700;
+  margin: 0;
+}
+
+.review {
+  display: flex;
+  margin: 0 16px 23px;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+}
+
+.review__flex {
+  display: flex;
+}
+
+.review__stars {
```

Comment:
You were supposed to use the previously implemented `stars` block but not rewrite it to be an element of some other block. Please change it.